integrationExamples/gpt/astraone_in-image_example.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
While I appreciate these integration examples to quickly test, I don't think you should include these intergrationExamples in the PR. It's not a norm that has been followed and is not necessary.

Reply:
Got it. Removed